src/components/map/MapLibre.tsx

```diff
@@ -58,14 +58,17 @@ type Props = {
   clusterDistance?: number;
   currentPosition?: LocationObject;
   geometryTourData?: LatLng[];
-  isLocationSelectable?: boolean;
   isMultipleMarkersMap?: boolean;
   isMyLocationButtonVisible?: boolean;
   locations: MapMarker[];
   mapCenterPosition?: LocationObjectCoords;
   mapStyle?: StyleProp<ViewStyle>;
   minZoom?: number;
-  onMapPress?: ({ nativeEvent }: { nativeEvent?: any }) => void;
+  onMapPress?: (arg0: {
+    geometry: {
+      coordinates: number[];
+    };
+  }) => Promise<{ isLocationSelectable?: boolean }>;
```

Comment:
`arg0` is weird, shouldn't this work?

```
onMapPress?: ({
    geometry: {
      coordinates: number[];
    };
  }) => Promise<{ isLocationSelectable?: boolean }>;
```

Reply:
I've updated what you sent a bit. It's more accurate this way. 

```
  onMapPress?: ({
    geometry: { coordinates }
  }: {
    geometry: {
      coordinates: number[];
    };
  }) => Promise<{ isLocationSelectable?: boolean }>;
```